src/librustdoc/html/static/themes/ayu.css

```diff
@@ -467,15 +460,38 @@ pre.rust .attribute .ident {}
 .content span.foreigntype,.content a.foreigntype,.block a.current.foreigntype {}
 pre.rust .doccomment {}
 .stab.deprecated {}
-.content .highlighted.attr,.content .highlighted.derive,.content .highlighted.macro {}
+.content a.attr,.content a.derive,.content a.macro {}
 .stab.portability {}
-.content .highlighted.union {}
 .content span.primitive,.content a.primitive,.block a.current.primitive {}
 .content span.externcrate,.content span.mod,.content a.mod,.block a.current.mod {}
-.content .highlighted.type {}
 pre.rust .kw-2,pre.rust .prelude-ty {}
 .content span.trait,.content a.trait,.block a.current.trait {}
 
+.search-results a:focus {}
```

Comment:
Maybe use comma instead of `{}` on every line and put the `{}` at the end?

Reply:
Ah nevermind, that would error. Is it normal that in ayu they're all the same?

Reply:
Yep, see the comment above (prior to FIXME; it's hidden by GitHub):

> /* rules that this theme does not need to set, here to satisfy the rule checker */